### PR TITLE
Version 0.14.0

### PR DESCRIPTION
**Pull request by florianduros**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+# [0.14.0] - 2022-01-05
+
+### Added
+
+- Web: Add removeLinks and getLinks method
+- Mobiles: Code block bindings
```
Comment (aringenbach) [inline CHANGELOG.md:8]: Looks like it's the first time we are "changelogging" something for both mobile platforms. I wonder if we want to go with "Mobile", "Mobiles" or "iOS/Android" but I guess we should make a choice now and stick to it.

Comment (florianduros) [reply CHANGELOG.md:8]: I'm in favour `ios/Android`, what do you think about it ?

Comment (jmartinesp) [reply CHANGELOG.md:8]: `iOS/Android` or `iOS+Android`, maybe.

Comment (florianduros) [reply CHANGELOG.md]: I replaced `Mobiles` by `iOS/Android`

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,18 @@
...
+# [0.14.0] - 2022-01-05
+
+### Added
+
+- Web: Add removeLinks and getLinks method
+- iOS/Android: Code block bindings
+
+### Fixed
+
+- Common: Code block fixes
```
Comment (jmartinesp) [inline CHANGELOG.md:12]: Actually, these should be "Code block & quote bindings/fixes".

Comment (florianduros) [reply CHANGELOG.md]: Fixed

Comment (jmartinesp) [reply CHANGELOG.md]: I actually meant 'Code block & quote bindings' in 'Added' and 'Code block & quote fixes' in 'Fixed', sorry 😅

Comment (jmartinesp) [reply CHANGELOG.md]: The bindings are only for iOS/Android, the fixes are common (they're in the Rust model).

Comment (florianduros) [reply CHANGELOG.md]: Okay, I fixed it

Comment (jmartinesp) [reply CHANGELOG.md]: Thanks! And sorry for the misunderstanding!

**Review by jmartinesp — APPROVED** [review]
LGTM!

**Pull request merged**